spec/models/book_spec.rb

```diff
@@ -85,7 +84,7 @@ def book_is_updated?
 
   describe '#populate!' do
     before do
-      @book = Book.add_task('0804720991')
+      @book = Book.new(isbn_10: '0804720991')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.